README.md

```diff
@@ -34,17 +34,17 @@ To update the installed JavaScript engines later on, just run `jsvu` again.
 
 ## Supported engines per OS
 
-| JavaScript engine         | Binary name               | `mac64`            | `win32` | `win64`            | `linux32` | `linux64` |
-| ------------------------- | ------------------------- | ------------------ | ------- | ------------------ | --------- | --------- |
-| [**Chakra**][ch]          | `chakra` or `ch`          | ✅                 | ✅      | ✅                 | ❌        | ✅        |
-| [**GraalJS**][graaljs]    | `graaljs`                 | ✅                 | ❌      | ✅                 | ❌        | ✅        |
-| [**Hermes**][hermes]      | `hermes` & `hermes-repl`  | ✅                 | ❌      | ✅                 | ❌        | ✅        |
-| [**JavaScriptCore**][jsc] | `javascriptcore` or `jsc` | ✅                 | ❌      | ✅ <sup>\*</sup>   | ❌        | ✅        |
-| [**QuickJS**][quickjs]    | `quickjs`                 | ❌                 | ✅      | ✅                 | ✅        | ✅        |
-| [**SpiderMonkey**][sm]    | `spidermonkey` or `sm`    | ✅                 | ✅      | ✅                 | ✅        | ✅        |
-| [**V8**][v8]              | `v8`                      | ✅                 | ✅      | ✅                 | ✅        | ✅        |
-| [**V8 debug**][v8]        | `v8-debug`                | ✅                 | ✅      | ✅                 | ✅        | ✅        |
-| [**XS**][xs]              | `xs`                      | ✅ <sup>(32)</sup> | ✅      | ✅ <sup>(32)</sup> | ✅        | ✅        |
+| JavaScript engine         | Binary name               | `mac64`            | `mac-arm64`     | `win32` | `win64`            | `linux32` | `linux64` |
```

Comment:
`mac-arm64` is the name that V8 bots use, but that's orthogonal to the name jsvu uses. I'd prefer sticking to `mac64arm` as the jsvu identifier (we already use it for JSC). WDYT?